config-model/src/main/java/com/yahoo/vespa/model/application/validation/QuotaValidator.java

```diff
@@ -33,9 +33,11 @@ public class QuotaValidator implements Validator {
     @Override
     public void validate(Context context) {
         var zone = context.deployState().zone();
-        var exclusivity = new Exclusivity(zone, context.deployState().featureFlags().sharedHosts());
-        var tuning = new CapacityPolicies.Tuning(context.deployState().featureFlags().adminClusterArchitecture(),
-                                                 context.deployState().featureFlags().logserverNodeMemory());
+        var featureFlags = context.deployState().featureFlags();
+        var exclusivity = new Exclusivity(zone, featureFlags.sharedHosts());
+        var tuning = new CapacityPolicies.Tuning(featureFlags.adminClusterArchitecture(),
+                                                 featureFlags.logserverNodeMemory(),
+                                                 featureFlags.logserverNodeMemory());
```

Comment:
did you really mean to use logserverNodeMemory twice here?

Reply:
No, this is wrong, thanks! Fixed